lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -25,14 +25,19 @@ import { UrlString } from "../url/UrlString";
 import { CcsCredentialType } from "../account/CcsCredential";
 import { buildClientInfoFromHomeAccountId } from "../account/ClientInfo";
 import { InteractionRequiredAuthError, InteractionRequiredAuthErrorMessage } from "../error/InteractionRequiredAuthError";
+import { IPerformanceClient } from "../telemetry/performance/IPerformanceClient"; 
+import { PerformanceEvents } from "../telemetry/performance/PerformanceEvent";
 
 /**
  * OAuth2.0 refresh token client
  */
 export class RefreshTokenClient extends BaseClient {
 
-    constructor(configuration: ClientConfiguration) {
+    protected performanceClient: IPerformanceClient;
```

Comment:
I think this should go into the `BaseClient`, so it is available everywhere. I think you'll need make it an optional parameter in the constructor for the `BaseClient` (and all of its subclasses).

Reply:
Ah nvm, can be a breaking change for msal-common.